test/jdk/java/awt/Frame/AwtFramePackTest.java

```diff
@@ -39,7 +39,6 @@
  * @test
  * @bug 8265586
  * @key headful
- * @requires (os.family == "windows")
  * @summary Tests whether insets are calculated correctly on Windows
  * for AWT Frame by checking the actual and expected/preferred frame sizes.
  * @run main/othervm -Dsun.java2d.uiScale=1 AwtFramePackTest
```

Comment:
Please double check that the "-Dsun.java2d.uiScale=1" option is needed here, It does not affect the frame decoration, and I think this can cause the test to fail if the desktop scale=200% is set in the WIndows.

Reply:
@mrserb Thank you for catching it :) 
I do see that the test fails when scale >= 200% and passes when the uiScale option is removed. I'll additionally test it on other platforms to check for stability before making the changes.

Reply:
@mrserb Removed uiScale=1,test works on all platforms.